Utils/github_workflow_scripts/get_xsoar_supported_release_notes.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+This script is used in the Review Release Notes GitHub workflow.
+It accepts a list of changed files (release notes)
+and returns the release notes of XSOAR-supported Packs.
+"""
+
+from typing import Dict, Any, List
+import json
+from pathlib import Path
+import argparse
+import sys
+import traceback
+import click
+
+
+def is_pack_xsoar_supported(pack_name: str) -> bool:
+    """
+    Checks whether the supplied Pack is from a XSOAR-supported
+
+    Args:
+        - `pack_name` (``str``): The name of the Pack
+
+    Returns:
+        `bool` representing whether Pack is XSOAR-supported or not
+    """
+
+    pack_metadata = Path("Packs", pack_name, "pack_metadata.json")
+
+    if pack_metadata.exists():
+        with pack_metadata.open(encoding="UTF-8") as pm:
+            metadata: Dict[str, Any] = json.load(pm)
```

Comment:
let's add try and except here

Reply:
To catch what? If the file exists? if the file can be parsed as JSON? 

In both cases I think that by the time we're in this stage of the flow, we're expecting the pack metadata to exist and be valid JSON since the SDK bumps the version in it and created the RN with the `update-release-notes` command. If the file doesn't exist or is not parsable as JSON, the `update-release-notes` command would fail and the release notes wouldn't get generated in the first place.

If we do still want to handle an exception, we're running `main` under `try..catch` block so I guess we would `raise` some type of exception according to what we're attempting to catch.